### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**
## Solution
- *[12.9.3](https://about.gitlab.com/releases/2020/04/14/critical-security-release-gitlab-12-dot-9-dot-3-released/)*
- *[12.9.4](https://about.gitlab.com/releases/2020/04/20/gitlab-12-9-4-released/)*
- *[12.10.0](https://about.gitlab.com/releases/2020/04/22/gitlab-12-10-released/)*
- *[12.10.1](https://about.gitlab.com/releases/2020/04/24/gitlab-12-10-1-released/)*
- *Enable Grafana*
- *Use the recommended nginx configuration*
- *Getting ready for Buster*

## PR Status
- [x] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/gitlab_ynh%20PR106%20(Kay0u)/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by kay0u** [comment]
Bump

**Comment by maniackcrudelis** [comment]
A lot of change in your code, don't you should call for a testing ?

**Comment by kay0u** [comment]
> A lot of change in your code, don't you should call for a testing ?

Done: https://forum.yunohost.org/t/gitlab-a-git-repository-manager/7499/7

**Comment by kay0u** [comment]
Ready for review?

File: scripts/upgrade.d/upgrade.11.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 gitlab_version="11.11.5"
 
+# There is no buster version for gitlab 11.X
+gitlab_x86_64_debian_verion="stretch"
```
Comment (maniackcrudelis) [inline scripts/upgrade.d/upgrade.11.sh:4]: What about `version` instead of `verion` ?

File: scripts/upgrade.d/upgrade.last.sh [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 package_check_action() {
 	local sysctl_file="$final_path/embedded/cookbooks/package/resources/gitlab_sysctl.rb"
-	if [ ! -f "$sysctl_file" ]; then
-		sysctl_file="$final_path/embedded/cookbooks/package/resources/sysctl.rb"
-	fi
-	ynh_replace_string --match_string="command \"sysctl -e --system\"" --replace_string="command \"sysctl -e --system || true\"" --target_file=$sysctl_file
+	ynh_replace_string --match_string="command \"sysctl -e \(.*\)\"" --replace_string="command \"sysctl -e \1 || true\"" --target_file=$sysctl_file
+	
+	sysctl_file="/opt/gitlab/embedded/cookbooks/package/recipes/sysctl.rb"
+	ynh_replace_string --match_string="command \"sysctl -e \(.*\)\"" --replace_string="command \"sysctl -e \1 || true\"" --target_file=$sysctl_file
```
Comment (maniackcrudelis) [inline scripts/upgrade.d/upgrade.last.sh:24]: It's not specific to package_check anymore?

File: scripts/upgrade.d/upgrade.11.sh (hunk 1 above)
Comment (kay0u) [reply scripts/upgrade.d/upgrade.11.sh]: indeed :+1:

File: scripts/upgrade.d/upgrade.last.sh (hunk 2 above)
Comment (kay0u) [reply scripts/upgrade.d/upgrade.last.sh:24]: No, with that fix you can install it on lxc (scaleway for example).

Here the related issue https://github.com/YunoHost-Apps/gitlab_ynh/issues/66

Comment (maniackcrudelis) [reply scripts/upgrade.d/upgrade.last.sh:24]: Ok, so finally you're just ignoring the error if it happens with sysctl.
And it does work anyway ?
Ok, that's fine for me.

Comment (kay0u) [reply scripts/upgrade.d/upgrade.last.sh:24]: Yes, it's working, but not as fast as possible if I well understand it. It's better not to ignore these errors, but in an environment where the kernel parameters are read-only, I couldn't find any other solution.

**Comment by kay0u** [comment]
Merging in 3 days

**Pull request merged**